Dockerfile

```diff
@@ -40,6 +43,7 @@ RUN apt install -y libxkbcommon0 \
     xvfb
 
 COPY ./deploy/requirements/* /opt/
+RUN curl -sS https://bootstrap.pypa.io/get-pip.py | python3.10
```

Comment:
why you install addition pip? If you installed it by

Reply:
I removed first pip installation
I get this error with it(when try install requirements):
ImportError: cannot import name 'html5lib' from 'pip._vendor' (/usr/lib/python3/dist-packages/pip/_vendor/__init__.py)